src/stub/parser.rs

```diff
@@ -276,25 +277,30 @@ impl<'a, I: Iterator<Item = &'a str>> Parser<I> {
         self.skip_to_next_line();
 
         let mut text_block: Vec<String> = Vec::new();
-        while let Some(line) = self.upto_newline() {
+        while let Some(line) = self.tokens_in_next_line() {
             text_block.push(line.join(" ").trim().to_string())
         }
 
         text_block.join("\n")
     }
 
     fn next_past_newline(&mut self) -> Option<&'a str> {
-        match self.stream.next() {
-            Some("\n") => self.stream.next(),
+        match self.next_token() {
+            Some("\n") => self.next_token(),
             Some("") => self.next_past_newline(),
             token => token,
         }
     }
 
+    fn next_token(&mut self) -> Option<&'a str> {
+        self.token_stream.next()
+    }
+
     // Consumes the newline
-    fn upto_newline(&mut self) -> Option<Vec<&'a str>> {
```

Comment:
The new name seems confusing, I would expect `tokens_in_next_line` to be the tokens after the line I'm currently in, and unless the functionality has changed, the function does return the tokens not after the next line, but from the current token until a newline.

Reply:
Fair enough, `tokens_upto_newline`?

Reply:
Sure